src/main/resources/logback.xml

```diff
@@ -13,9 +13,9 @@
         </encoder>
     </appender>
 
-    <root level="info">
+    <root level="debug">
```

Comment:
This should be changed back to `INFO`, or perhaps `WARN` or `ERROR`, and additional package-specific levels set as needed, e.g.:

```xml
   <logger name="io.spicelabs" level="debug" />
```

Reply:
Alternatively/additionally, a second `logback.xml` could be added to `src/test/resources/` to set noisier levels for testing.